desktop/core/ext-py3/boto-2.49.0/boto/utils.py

```diff
@@ -1030,7 +1030,7 @@ def compute_md5(fp, buf_size=8192, size=None):
 
 
 def compute_hash(fp, buf_size=8192, size=None, hash_algorithm=md5):
-    hash_obj = hash_algorithm(usedforsecurity=False) if hash_algorithm == md5 else hash_algorithm()
+    hash_obj = hashlib_new('md5', usedforsecurity=False) if hash_algorithm == md5 else hash_algorithm()
```

Comment:
Perhaps add a comment here given that we're changing a third-party module.

Would this be fixed in a boto upgrade, if not would it be worth opening a PR there as well?

Reply:
Thanks for reviewing, I've added a comment.

Boto2 is very old and not maintained anymore, so opening PR in that project wont work. Long-term solution is to upgrade to boto3 which is a roadmap item and support that with FIPS.